File: src/main/java/de/nilzbu/demo/domain/dayvalues/DayValues.java
package de.nilzbu.demo.domain.dayvalues;


import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import org.hibernate.annotations.JdbcTypeCode;
import org.hibernate.type.SqlTypes;

import java.util.Date;
import java.util.UUID;

@Entity
@Getter
@Setter
public class DayValues {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Column(name = "id", nullable = false)
    @JdbcTypeCode(SqlTypes.VARCHAR)
    private UUID id;

    private Date date;
    private int sys;
    private int dia;
    private int pulse;
    private double weight;
}

File: frontend/src/app/dto/dayValues/dayValues.ts
export interface DayValues {
  id: string;
  date: Date;
  sys: number;
  dia: number;
  pulse: number;
  weight: number;
}

export interface DayValuesWithoutId {
  date: Date;
  sys: number;
  dia: number;
  pulse: number;
  weight: number;
}

export interface DayValueId {
  id: String
}
